x/vesting/keeper/msg_server.go

```diff
@@ -199,15 +195,14 @@
 	ak := k.accountKeeper
 	bk := k.bankKeeper
 
-	// NOTE: ignore error in case dest address is not defined
-	dest, _ := sdk.AccAddressFromBech32(msg.DestAddress)
-
-	// NOTE: error checked during msg validation
+	// NOTE: errors checked during msg validation
 	addr := sdk.MustAccAddressFromBech32(msg.AccountAddress)
+	funder := sdk.MustAccAddressFromBech32(msg.FunderAddress)
 
-	// Default destination to funder address
+	// NOTE: ignore error in case dest address is not defined and default to funder address
+	dest, _ := sdk.AccAddressFromBech32(msg.DestAddress)
```

Comment:
should we check the error and then only default to the FunderAddress if it is equal to an empty string? Because if there is something else wrong, the user might not realize it was defaulted to the funder address. 🤔